### PR TITLE
[1LP][RFR] Added test_alert_run_ansible_playbook

### PR DESCRIPTION
**Pull request by quarckster**
{{pytest: -v -k "test_alert_run_ansible_playbook" --long-running}}

### CODE REVIEW

**Comment by rlbabyuk** [comment]
@quarckster Please rebase

File: cfme/tests/ansible/test_embedded_ansible_automate.py [hunk 1]
```diff
@@ -225,18 +292,26 @@ def test_ansible_playbook_button_crud(ansible_catalog_item, appliance, request):
...
-        assert view.provisioning.details.get_text_of("Hosts") == full_template_vm.ip_address
+        assert hosts == full_template_vm_modscope.ip_address
     assert view.provisioning.results.get_text_of("Status") == "successful"
+
+
+def test_alert_run_ansible_playbook(full_template_vm_modscope, alert_profile, request, appliance):
+    full_template_vm_modscope.add_tag("Service Level", "Gold")
+    full_template_vm_modscope.remove_tag("Service Level", "Gold")
+    request.addfinalizer(lambda: appliance.ssh_client.run_command(
+        "if [ -f \"/var/tmp/modified-release\" ]; then rm \"/var/tmp/modified-release\""))
```
Comment (izapolsk) [inline cfme/tests/ansible/test_embedded_ansible_automate.py]: OPTIONAL: I'd advice you to use ' instead " to make code more neat.

Comment (izapolsk) [inline cfme/tests/ansible/test_embedded_ansible_automate.py]: REQUIRED: this code won't work you at least don't have 'fi' at the end;
And I'm under impression that rm -f would suit much better in this case.
If you persist you can use something like the following
[[ -f blabla ]] && rm blabla

**Comment by dajoRH** [comment]
Lint report for commit a03ac610352679113598cdb8803b31dcc35f508f:

`cfme/tests/ansible/test_embedded_ansible_automate.py`:
- :heavy_exclamation_mark: Line 324:69: **W292** *no newline at end of file*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Pull request merged**